src/system/WatchableEventManagerLibevent.h

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file declares an implementation of WatchableEvents using libevent.
```

Comment:
nit: do we need this `@file` description given that the file name is already "Watchable Event Manager Libeevent" ?

Reply:
We don't, but this particular line is not actually touched in the PR, it's just carried over from the parent file this was split from ([`WatchableSocketLibevent.h` → `WatchableEventManagerLibevent.h`](https://gist.github.com/kpschoedel/451cf083e707c2a5405267dca5384f2a))